csob/api.py

```diff
@@ -0,0 +1,290 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import List, Optional, Union
+
+from csob.enums import (
+    Currency, HTTPMethod, Language, PaymentButtonBrand, PayMethod, PayOperation, ResultCode
+)
+from csob.payment import Item
+
+
+AmountHundredths = Union[Decimal, int]
+
+
+class APIResponse:
+    response_json: dict
+    date_time: datetime
+    signature: str
+    result_code: ResultCode
+
+    def is_okay(self) -> bool:
+        """
+        Check if result code is OK or 8*0.
```

Comment:
tomu moc nerozumím

Reply:
8*0 je 0

Reply:
@matllubos to je myšleno jako regex jakože 800 nebo 810 apod.

Reply:
tak to má být 8.0 :D * je 0 a více znaků

Reply:
nebo raději dej 8[0-9]0

Reply:
pravd :D